src/validation.js

```diff
@@ -1,9 +1,11 @@
-const VALID_EVENT_NAME = 'pull_request';
+const VALID_EVENT_NAMES = ['pull_request','pull_request_target'];
 
 const validateEnv = (github) => {
   const { eventName } = github.context;
-  if (eventName === VALID_EVENT_NAME) return;
-  const error = `This action runs only in the "${VALID_EVENT_NAME}" event. Change the property "on" of your workflow file from "${eventName}" to "${VALID_EVENT_NAME}".`;
+  for(let validEvent of VALID_EVENT_NAMES){
+    if(eventName === validEvent) return;
+  }
+  const error = `This action runs only in one of the following events "${VALID_EVENT_NAMES.join()}". Change the property "on" of your workflow file from "${eventName}" one of "${VALID_EVENT_NAME.join()}".`;
```

Comment:
- Improve readability.
- Fix text (add `to` word)
```suggestion
  const validEvents = VALID_EVENT_NAMES.map(e => `"${e}"`).join(', ');
  const error = `This action runs only in one of the following events: ${validEvents}. Change the property "on" of your workflow file from "${eventName}" to one of ${validEvents}.`;
```